android/engine/src/test/java/org/smartregister/fhircore/engine/cql/LibraryEvaluatorTest.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.cql
+
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileInputStream
+import java.io.IOException
+import java.io.InputStreamReader
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+class LibraryEvaluatorTest {
+  var libraryEvaluator: LibraryEvaluator? = null
+  var libraryData = ""
+  var helperData = ""
+  var valueSetData = ""
+  var testData = ""
+  var result = ""
+  var evaluatorId = "ANCRecommendationA2"
+  var context = "Patient"
+  var contextLabel = "mom-with-anemia"
+  @Before
+  fun setUp() {
+    try {
+      libraryData = readJsonFile(ASSET_BASE_PATH + "library.json")
+      helperData = readJsonFile(ASSET_BASE_PATH + "helper.json")
+      valueSetData = readJsonFile(ASSET_BASE_PATH + "valueSet.json")
+      testData = readJsonFile(ASSET_BASE_PATH + "patient.json")
+      result = readJsonFile(ASSET_BASE_PATH + "result.json")
+    } catch (e: IOException) {
+      e.printStackTrace()
+    }
+  }
+
+  @Test
+  fun runCql() {
+    libraryEvaluator = LibraryEvaluator()
+    val auxResult =
```

Comment:
It is preferable to have the _full variable name_ instead since it is self documenting

Reply:
@ndegwamartin Ok, will work on it.